docs/ASM.md

```diff
@@ -174,12 +174,12 @@ Note that the load balancing set will be created if it does not exist. If it exi
 #### Options for Bootstrapping a Windows Node in Azure
 
     :bootstrap_protocol            Default is winrm for a windows image
-    :winrm_password                The WinRM password
-    :winrm_authentication_protocol Defaults to negotiate, supports kerberos, can be set to basic for debugging
-    :winrm_transport               Defaults to plaintext, use ssl for improved privacy
-    :winrm_port                    Defaults to 5985 plaintext transport, or 5986 for SSL
+    :connection_password           The WinRM password
+    :winrm_auth_method             Defaults to negotiate, supports kerberos, can also be set to plaintext or ssl
```

Comment:
In knife bootstrap they have this description The WinRM authentication method to use. (valid options: 'plaintext', 'kerberos', 'ssl', or 'negotiate'). May be we can match with it.